AnnService/inc/Core/BKT/Index.h

```diff
@@ -104,6 +105,7 @@ namespace SPTAG
             inline IndexAlgoType GetIndexAlgoType() const { return IndexAlgoType::BKT; }
             inline VectorValueType GetVectorValueType() const { return GetEnumValueType<T>(); }
             
+            inline bool IsIdenticalVector(const void* pX, const void* pY, float eps) const { return COMMON::DistanceUtils::ComputeL2Distance((const T*)pX, (const T*)pY, m_pSamples.C()) < eps; }
```

Comment:
will sptag only support l2? how about cosine distance?

Reply:
cosine is also supported. It can be more accurate using L2 to judge whether two vectors are the same.

Reply:
the meaning of the function is “identical" upon specified space instead of exact match, like for cosine distance, we store normalized vector, but the input vector may be unnormalized, so we should pick corresponding distance function.

Reply:
ok, let me change it.